Source/Core/DolphinQt2/Config/Graphics/GraphicsWindow.h

```diff
@@ -0,0 +1,33 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <QDialog>
+
+class QTabWidget;
+class QDialogButtonBox;
+
+namespace X11Utils
+{
+class XRRConfiguration;
+}
+
+class GraphicsWindow final : public QDialog
+{
+  Q_OBJECT
+public:
+  explicit GraphicsWindow(QWidget* parent, X11Utils::XRRConfiguration* xrr_config);
```

Comment:
1) Yes, good idea
2) 
* It can't be created inside; could have some unwanted side-effects AFAICT
* It probably has to be passed in as the ``GetAvailableResolutions()`` method needs it and it can't be constructed inside

Maybe there's a better way that I'm not seeing here, so I'm always open for suggestions.